index.html

```diff
@@ -1,413 +1,275 @@
-              ></a>
-            </li>
-          </ul>
-        </article>
-        <article class="coach">
-          <h3 class="coach__title">Nico Koenig</h3>
-          <img
-            src="images/coach_nico.png"
-            class="coach__image"
-            alt="Nico Koenig"
-          />
-          <p class="coach__bio">
-            Nico is a freelancer and trainer for HTML, CSS & JavaScript. In his
-            non-existing free time, Nico organizes the Franken JS meetup and
-            brews some kickass hand ground coffee.
-          </p>
-          <ul class="coach__social">
-            <li>
-              <a
-                href="https://twitter.com/theNicoKoenig"
-                target="_blank"
-                rel="noopener noreferrer"
-                title="Follow Nico on twitter"
-                ><i class="icon-twitter-squared"></i
-              ></a>
-            </li>
-          </ul>
-        </article>
+
+      <section class="coach">
+        <h3 class="coach__title">Miroslav Jonas</h3>
+        <img src="images/coach_miro.jpeg" class="coach__image" alt="Miro" />
+        <p class="coach__bio">
+          Miro is a software developer interested in the frontend of things, speaker, organizer of Angular Vienna and
+          ViennaJS meetups and open source enthusiast. He likes spending free time outdoors, wandering the world,
+          with or without his camera.
+        </p>
+        <ul class="coach__social">
+          <li>
+            <a href="https://twitter.com/meeroslav" target="_blank" rel="noopener noreferrer"
+              title="Follow Miro on twitter"><i class="icon-twitter-squared"></i></a>
+          </li>
+        </ul>
       </section>
-      <section>
-        <h2>Location and date</h2>
-        <p>
-          The Event will happen on 29th & 30th of July 2019. The Event is
-          scheduled from 9am to 5pm.
+
+      <section class="coach">
+        <h3 class="coach__title">Adrián Bolonio</h3>
+        <img src="images/coach_adrian.png" class="coach__image" alt="Adrián Bolonio" />
+        <p class="coach__bio">
+          Adrián is an experienced Web Team Lead, Frontend Developer, and Web Accessibility (a11y) advocate. He's from
+          Spain, but since 2012 he has been living and working in Vienna. When he's not at the office he enjoys a good
+          read, working his way through any delicious recipe, and indulging his love for travelling to new places.
         </p>
-        <p>
-          ZOLLHOF - Tech Incubator<br />
-          Kohlenhofstraße 60<br />
-          90443 Nürnberg<br />
-          Germany<br />
+        <ul class="coach__social">
+          <li>
+            <a href="https://twitter.com/bolonio" target="_blank" rel="noopener noreferrer"
+              title="Follow Adrián on twitter"><i class="icon-twitter-squared"></i></a>
+          </li>
+        </ul>
+      </section>
+
+      <section class="coach">
+        <h3 class="coach__title">Rob Axelson</h3>
```

Comment:
Typo: `Axelson` should be `Axelsen` :smile: 

For sake of consistency, could you please also make it `Robert Axelsen` in title, and simply `Rob` in description?: `Rob is a front end developer...`

Reply:
LGTM now. Thanks @meeroslav